js/admin.vue

```diff
@@ -1,9 +1,11 @@
 <template>
+<div>
```

Comment:
Could the whole block be indented once more? (as well as in the other components where a root div has been added)